apps/platform/src/components/dashboard/variable/variableCard/index.tsx

```diff
@@ -63,11 +64,14 @@ export default function VariableCard(
           <AccordionTrigger
             className="hover:no-underline"
             rightChildren={
-              <div className="text-xs text-white/50">
+              <div className="flex items-center gap-x-4 text-xs text-white/50">
                 {dayjs(variable.updatedAt).toNow(true)} ago by{' '}
-                <span className="text-white">
-                  {variable.lastUpdatedBy.name}
-                </span>
+                <div className="flex items-center gap-x-2">
+                  <span className="text-white">
+                    {variable.lastUpdatedBy.name}
+                  </span>
+                  <AvatarComponent name={variable.lastUpdatedBy.name} />
```

Comment:
I think you are also supposed to pass the image url

Reply:
I think you should do it as it is done in `apps/platform/src/components/shared/navbar/index.tsx`